package org.example.test_tasks_whiletrue.exception.userExceptions;

public class UserNotFoundException extends Exception {
    public UserNotFoundException(String message) {
        super(message);
    }
}
